src/__tests__/__snapshots__/SideNav.tsx.snap

```diff
@@ -87,7 +87,7 @@ exports[`SideNav SideNav.Link renders consistently 1`] = `
 
 .c1.variant-normal > .c0[aria-current='page']::before,
 .c1.variant-normal > .c0[aria-selected='true']::before {
-  background-color: #f66a0a;
+  background-color: #f9826c;
```

Comment:
Wasn't quite sure if this was okay or not - my understanding was that implementing these functional color variables would not change the presentation.

<img width="272" alt="Screen Shot 2021-02-25 at 11 31 52 AM" src="https://user-images.githubusercontent.com/16739070/109192903-6c71e700-775d-11eb-971d-b9990dd5c5e8.png">

This change comes from using `colors.sidenav.borderActive` which seems like the obvious functional choice here and is very close to the original color. Thoughts on what we should do here?